pkg/operator/deploy/deploy.go

```diff
@@ -138,6 +138,15 @@ func (o *operator) resources() ([]kruntime.Object, error) {
 		domain += "." + o.env.Domain()
 	}
 
+	ingressIP := o.oc.Properties.IngressProfiles[0].IP
```

Comment:
`o.oc.Properties.IngressProfiles` can be `nil`, I think.

Reply:
Fixed, Thanks